lua/rest-nvim/request/init.lua

```diff
@@ -67,23 +66,6 @@ local function get_body(bufnr, start_line, stop_line, has_json)
     end
   end
 
-  local is_json, json_body = pcall(vim.fn.json_decode, body)
-
-  if is_json then
-    if has_json then
-      -- convert entire json body to string.
-      return vim.fn.json_encode(json_body)
-    else
-      -- convert nested tables to string.
-      for key, val in pairs(json_body) do
-        if type(val) == "table" then
-          json_body[key] = vim.fn.json_encode(val)
-        end
-      end
-      return json_body
-    end
-  end
-
```

Comment:
DONOT delete these lines; this statement does two things:
1. LINE 75: Minify json. ie. remove extra tabs and spaces to reduce bandwidth.
2. LINE 78-83: Parse nested json into valid `application/x-www-form-urlencoded` string values. see #138